orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -106,7 +106,6 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
     }
 
     if (strategySupportsRollback) {
-      // TODO: should this append after any existing stages in the graph?
```

Comment:
The answer was yes ... I didn't see anything on `StageGraphBuilder` that would let you `connect()` to the last added node.

Reply:
Yeah, I think `StageGraphBuilder` could be a little easier to use for certain common scenarios.